src/pybind/CMakeLists.txt

```diff
@@ -67,19 +67,39 @@ add_custom_target(copy_python_files ALL DEPENDS ${NMODL_PYTHON_FILES_OUT})
 add_custom_command(
   OUTPUT ${NMODL_PYTHON_FILES_OUT}
   COMMAND ${CMAKE_COMMAND} -E copy_directory ${NMODL_PROJECT_SOURCE_DIR}/nmodl
-          ${PROJECT_BINARY_DIR}/lib/python/nmodl
+          ${PROJECT_BINARY_DIR}/lib/nmodl
   COMMAND ${CMAKE_COMMAND} -E copy_if_different $<TARGET_FILE:_nmodl> ${PROJECT_BINARY_DIR}/nmodl
+  COMMAND
+    if test -f "${PROJECT_BINARY_DIR}/lib/nmodl/libpywrapper.so" \; then mv
+    ${PROJECT_BINARY_DIR}/lib/nmodl/libpywrapper.so ${PROJECT_BINARY_DIR}/lib \; fi
```

Comment:
Can you check extension for OS X?

Reply:
this line is old. Thank you for the rest because I was going to forget. I will add wheel tests for mac too (as soon as it is working)